src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -420,4 +422,16 @@ private function checkUserAction(Entry $entry)
             throw $this->createAccessDeniedException('You can not access this entry.');
         }
     }
+
+    /**
+     * Check for existing entry, if it exists, redirect to it with a message.
+     *
+     * @param $entry
+     *
+     * @return array|bool
+     */
+    public function checkIfEntryAlreadyExists($entry)
```

Comment:
Oh yes it was what I wanted to do (but phpstorm refactored with a public method ... and I didn't checked!)